src/_ert_forward_model_runner/runner.py

```diff
@@ -23,14 +28,42 @@ def __init__(self, jobs_data):
 
         self._set_environment()
 
+    def _manifest_dict(self):
+        with open("manifest.json", mode="r", encoding="utf-8") as f:
+            data = json.load(f)
+        timeout = 10
+        manifest = {}
+        keep_waiting = True
+        while keep_waiting:
+            keep_waiting = False
+            for name, file in data.items():
+                path = Path(file)
+                if name not in manifest:
+                    manifest[name] = {
+                        "type": "file",
+                        "path": str(path.absolute()),
+                    }
+                path_exists = path.exists()
+                if path_exists and "md5sum" not in manifest[name]:
+                    md5sum = hashlib.md5(path.read_bytes()).hexdigest()
+                    manifest[name]["md5sum"] = md5sum
+                elif not path_exists:
+                    timeout -= 1
```

Comment:
`timeout` seems more like the number of retries :)

Reply:
Removed this logic after a discussion with @eivindjahren